runtime/src/main/resources/application.properties

```diff
@@ -50,4 +50,5 @@ huaweicloud.auto.approve.service.template.enabled=false
 flexibleengine.auto.approve.service.template.enabled=false
 openstack.auto.approve.service.template.enabled=false
 scs.auto.approve.service.template.enabled=false
-region.azs.cache.expire.time.in.minutes=60
\ No newline at end of file
+region.azs.cache.expire.time.in.minutes=60
+huaweicloud.sdk.debug.http.listener.enabled=false
```

Comment:
i think we can rename it to - huaweicloud.sdk.enable.http.debug.logs

Reply:
Updated.